lib/dbservice/constants/mappings.ex

```diff
@@ -315,22 +315,27 @@ defmodule Dbservice.Constants.Mappings do
       optional: [],
       type: :date
     },
-    "auth_group" => %{db_field: "auth_group", required: ["student"], optional: [], type: :string},
+    "auth_group" => %{
+      db_field: "auth_group",
+      required: ["student", "teacher_creation"],
```

Comment:
```suggestion
      required: ["student", "teacher_addtion"],
```
Or something more appropriate